docs/release_notes.md

```diff
@@ -1,5 +1,39 @@
 # Release Notes
 
+## Version 1.1.0
+
+### Bug Fixes
+
+- Filtered characters to get valid names for nodes and attributes since some special characters are not allowed in Maya for Linux. *AdonisFX-1033*
+- Fixed Python runtime error due to incorrect element selection on node creation. *AdonisFX-1031*
+- Fixed error messages not displaying the proper status of the AdnSimshape connections. *AdonisFX-1034*
+- Fixed exporter creating the AAD file when there is nothing to export or the only thing to export failed. *AdonisFX-1032*
+- Fixed manipulation of referenced nodes while using the AdonisFX paint tool. *AdonisFX-1147*
+- Fixed a bug that did not allow to add attachments on a AdnMuscle/AdnRibbonMuscle deformer created on referenced geometry. *AdonisFX-506*
+- Fixed a bug in the handling of the constraint weight arrays. *AdonisFX-1155*
+- Fixed a bug that caused geometry sliding constraints to not find valid triangles to slide on for some vertices in very specific scenarios. *AdonisFX-1165*
+- Fixed flood operation not working correctly for multi influence constraints. *AdonisFX-1146*
+
+### Improvements
+
+- Optimzed the initialization of the debugger improving the performance on scenes with many AdonisFX deformers. *AdonisFX-1167*
```

Comment:
There are some things to say there.
1. This improvement was removed as the method where it was had a lot of complexity that got reduced drastically by the new utility. So, I would remove this line.
2. We added a new utility to AdonisFX menu to help with the debugging graph connections. #1171. That is not present. Here I would mention the utility and improvement.
3. "Optimized"

Reply:
1. Removed this improvement.
2. Added #1171 (I think it was not merged when I did the first list) as improvement. However the issue is tagged as feature, should we move it to feature?

Reply:
We should move it to features, yes!

Reply:
Moved to features.